src/Main.cpp

```diff
@@ -180,7 +180,7 @@ std::unique_lock<std::mutex> Main::get_lock_converters() {
 
 /// \brief Main program loop.
 ///
-/// Start conversion worker threads, poll for command sfrom Kafka.
+/// Start conversion worker threads, poll for commands from Kafka.
 /// When stop flag raised, clear all workers and streams.
 void Main::forward_epics_to_kafka() {
   using CLK = std::chrono::steady_clock;
```

Comment:
`CLK` is used outside of this function in the same way, could this be defined in the header?

Reply:
Nothing to do with this PR, but if you want to change it please feel free.

Reply:
ok no problem. i'll merge this now